server/tserver/src/main/java/org/apache/accumulo/tserver/FileManager.java

```diff
@@ -287,7 +290,15 @@ private void closeReaders(Collection<FileSKVIterator> filesToClose) {
     Map<FileSKVIterator,String> readersReserved = new HashMap<>();
 
     if (!tablet.isMeta()) {
+      long start = System.currentTimeMillis();
       filePermits.acquireUninterruptibly(files.size());
+      long waitTime = System.currentTimeMillis() - start;
+
+      if (waitTime >= slowFilePermitMillis) {
+        String message = new StringBuilder(128).append("Slow file permits request: ")
+            .append(waitTime).append(" ms, files requested: ").append(files.size()).toString();
+        log.info(message);
```

Comment:
This could iuse slf4j substitution instead of the string builder.  May be useful to include the key extent in the log message so the tablet is known.

Any particular reason for logging at info? I don't think the level should be changed, just curious.

Reply:
TSERV_SLOW_FLUSH_MILLIS logs at INFO so I was trying to follow the lead there. I will add the key extent and convert to slf4j sbustitution